group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorService.java

```diff
@@ -595,27 +568,17 @@ public CompletableFuture<List<ConsumerGroupDescribeResponseData.DescribedGroup>>
                     "consumer-group-describe",
                     topicPartition,
                     (coordinator, lastCommittedOffset) -> coordinator.consumerGroupDescribe(groupIds, lastCommittedOffset)
-                ).exceptionally(exception -> {
-                    if (!(exception instanceof KafkaException)) {
-                        log.error("ConsumerGroupDescribe request {} hit an unexpected exception: {}.",
-                            groupList, exception.getMessage());
-                    }
-
-                    return ConsumerGroupDescribeRequest.getErrorDescribedGroupList(
-                        groupList,
-                        Errors.forException(exception)
-                    );
-                });
+                ).exceptionally(exception -> handleOperationException(
+                    "ConsumerGroupDescribe",
+                    groupList,
+                    exception,
+                    (error, __) -> ConsumerGroupDescribeRequest.getErrorDescribedGroupList(groupList, error)
+                ));
 
             futures.add(future);
         });
 
-        final CompletableFuture<Void> allFutures = CompletableFuture.allOf(futures.toArray(new CompletableFuture[0]));
```

Comment:
Is line 581 just a one line for these lines? EDIT: I see the helper now 👍

Reply:
Yeah. Nice, isn't it? ;)